Source/ToolCore/Build/BuildWindows.cpp

```diff
@@ -152,6 +152,12 @@ void BuildWindows::Build(const String& buildPath)
     if (buildFailed_)
         return;
    
+    if (!BuildCreateDirectory(buildPath_ + "/Settings"))
+        return;
+
+    if (!BuildCopyFile(GetSettingsDirectory() + "/Engine.json", buildPath_ + "/Settings/Engine.json"))
```

Comment:
This is great too, settings on deployment, this is pretty simple/powerful really...